bolt_store.go

```diff
@@ -194,6 +195,15 @@ func (b *BoltStore) StoreLogs(logs []*raft.Log) error {
 
 	metrics.AddSample([]string{"raft", "boltdb", "logsPerBatch"}, float32(len(logs)))
 	metrics.AddSample([]string{"raft", "boltdb", "logBatchSize"}, float32(batchSize))
+	// Both the deferral and the inline function are important for this metrics
+	// accuracy. Deferral allows us to calculate the metric after the tx.Commit
+	// has finished and thus account for all the processing of the operation.
+	// The inlined function ensures that we do not calculate the time.Since(now)
+	// at the time of deferral but rather when the go runtime executes the 
+	// deferred function.
+	defer func() {
+		metrics.AddSample([]string{"raft", "boltdb", "writeCapacity"}, (float32(1000000000) / float32(time.Since(now).Nanoseconds())) * float32(len(logs)))
```

Comment:
Maybe digit separators e.g. 1_000_000_000

Reply:
Done (also required bumping the version in go.mod). I took it to 1.16 as that is the oldest supported Go version.